tensorflow/core/kernels/bias_grad_ali_op_cpu.h

```diff
@@ -212,15 +209,19 @@ void MultipleColumnWiseReduction(const float* A, int m, int n, int lda,
       }
 
       if (remain_num != 0) {
-        _mm512_mask_storeu_ps(Y + offset_block_end, mask, zero);
+        for (int j = offset_block_end; j < n; ++j) {
```

Comment:
For L205-L215, could use faster implement: memset(Y, 0, n*sizeof(float)). And remove L200-L203.